tests/testonto/excelparser/fromexcelonto.ttl

```diff
@@ -1,17 +1,17 @@
 @prefix : <http://emmo.info/emmo/domain/onto#> .
-@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix core: <http://www.w3.org/2004/02/skos/core#> .
 @prefix emmo: <http://emmo.info/emmo#> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
-@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix term: <http://purl.org/dc/terms/> .
```

Comment:
dcterms is the common prefix

Reply:
Yes, this is strange. I have no idea why this suddenly happens. I will have to check.

Reply:
I guess it comes from Owlready2 - it uses the second last component in the IRI as a sort of prefix.